src/assets/styles/base/_layout.scss

```diff
@@ -47,22 +47,58 @@ main {
 .section--margin-b {
   margin-bottom: 0;
 
-  @media screen and (min-width: map-get($bp, 'md')) {
+  @media screen and (min-width: map-get($breakpoint, "breakpoint-3")) {
     margin-bottom: 6em;
   }
 
   &--sm-spacing {
-    margin-bottom: $spacing-vertical-lg;
+    margin-bottom: $spacing-6;
 
-    @media screen and (min-width: map-get($bp, 'lg')) {
-      margin-bottom: $spacing-vertical-xl;
+    @media screen and (min-width: map-get($breakpoint, "breakpoint-4")) {
+      margin-bottom: $spacing-8;
     }
   }
 }
 
 .blog-post {
-  @media screen and (min-width: map-get($bp, 'xl')) {
-    padding-right: 60px;
-    padding-bottom: 60px;
+  @media screen and (min-width: map-get($breakpoint, "breakpoint-6")) {
+    padding-right: $spacing-8 - .75rem;
```

Comment:
This could be $spacing-7 (3rem) instead. The above code means that the current value is around 3.25rem; pretty pointless - not too sure why I left that in... It's now resolved.